testing/README.md

```diff
@@ -304,7 +304,7 @@ def test_relation_data():
         ),
     }
 
-# which is very idiomatic and superbly explicit. Noice.
+# which is very idiomatic and superbly explicit.
```

Comment:
This is funny

Reply:
It made me sad to remove it, but codespell complained and it seemed silly to keep it and put in a codespell bypass just for this.

Reply:
hahaha I think I added that during an early live demo, too bad it has to go :)